deploy/selfhost/install.sh

```
@@ -204,6 +215,10 @@
     echo "Begin Installing Plane"
     echo ""
 
+    if [ "$APP_RELEASE" == "stable" ]; then
+        export APP_RELEASE=$(checkLatestRelease)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)